connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -125,15 +125,14 @@
     public static final String ERRORS_TOLERANCE_NONE = "none";
     public static final String ERRORS_TOLERANCE_ALL = "all";
     public static final String ERRORS_TOLERANCE_DEFAULT = ERRORS_TOLERANCE_NONE;
-    public static final String ERRORS_TOLERANCE_DOC = "Behavior for tolerating errors during connector operation. NONE is the default " +
-            "and signals that any error will result in a connector task failure, and this matches the old behavior. ALL changes the " +
-            "behavior to skip over problem records.";
+    public static final String ERRORS_TOLERANCE_DOC = "Behavior for tolerating errors during connector operation. 'NONE' is the default value " +
+            "and signals that any error will result in an immediate connector task failure. 'ALL' changes the behavior to skip over problematic records.";
```

Comment:
Aren't the actual values that are used in the recommender are lowercase? If so, we should be consistent here.

Reply:
Either work (we ignore the case). but I'll update the code though.

Reply:
In that case, we can just reuse the enum instead of defining another string constant?